Tests/SourceGenerators/SourceGeneratorAssemblyToProcess/Generated/Realm.SourceGenerator/Realms.SourceGenerator.RealmGenerator/AllTypesClass_generated.cs

```diff
@@ -0,0 +1,259 @@
+﻿// <auto-generated />
+using SourceGeneratorPlayground;
+using SourceGeneratorPlayground.Generated;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+using System.Xml.Serialization;
+using System.Reflection;
+using System.ComponentModel;
+using Realms;
+using Realms.Weaving;
+using Realms.Schema;
+
+namespace SourceGeneratorPlayground
+{
+    [Generated]
+    [Woven(typeof(AllTypesClassObjectHelper))]
+    public partial class AllTypesClass : IRealmObject, INotifyPropertyChanged, IReflectableType
+    {
+        public static ObjectSchema RealmSchema = new ObjectSchema.Builder("AllTypesClass", ObjectSchema.ObjectType.RealmObject)
+        {
+            Property.Primitive("CharProperty", RealmValueType.Int, isPrimaryKey: false, isIndexed: false, isNullable: false, managedName: "CharProperty"),
+        }.Build();
+
+        #region IRealmObject implementation
+
+        private IAllTypesClassAccessor _accessor;
+
+        IRealmAccessor IRealmObjectBase.Accessor => Accessor;
+
+        internal IAllTypesClassAccessor Accessor => _accessor = _accessor ?? new AllTypesClassUnmanagedAccessor(typeof(AllTypesClass));
+
+        [IgnoreDataMember, XmlIgnore]
+        public bool IsManaged => Accessor.IsManaged;
+
+        [IgnoreDataMember, XmlIgnore]
+        public bool IsValid => Accessor.IsValid;
+
+        [IgnoreDataMember, XmlIgnore]
+        public bool IsFrozen => Accessor.IsFrozen;
+
+        [IgnoreDataMember, XmlIgnore]
+        public Realm Realm => Accessor.Realm;
+
+        [IgnoreDataMember, XmlIgnore]
+        public ObjectSchema ObjectSchema => Accessor.ObjectSchema;
+
+        [IgnoreDataMember, XmlIgnore]
+        public DynamicObjectApi DynamicApi => Accessor.DynamicApi;
+
+        [IgnoreDataMember, XmlIgnore]
+        public int BacklinksCount => Accessor.BacklinksCount;
+
+        public void SetManagedAccessor(IRealmAccessor managedAccessor, IRealmObjectHelper helper = null, bool update = false, bool skipDefaults = false)
+        {
+            var newAccessor = (IAllTypesClassAccessor)managedAccessor;
+            var oldAccessor = _accessor as IAllTypesClassAccessor;
+            _accessor = newAccessor;
+
+            if (helper != null)
+            {
+
```

Comment:
We should clean that up

Reply:
This is a more general problem of using string interpolation with substrings that could be empty or not. 
If we want to avoid all string we could build the whole method using string builder with append/appendLine, but the code (our code) will become less readable in my opinion. Do you have any alternative suggestion?